### ISSUE TITLE
Invalid authentication

### ISSUE THREAD

**Issue by HydrelioxGitHub** [drakhart/ha-super-soco-custom#4]
Hello,
I don't know where to find log, please point at me where I can find it to provide them here.

Anyway I can log into the app and not into the integration using the same credentials. The only difference is that I can't put a 0 at the begining of the phone number. I suppose this is the problem maybe...

Excuse me for my poor English as it's not my naative language.
Thank you for your work.

**Comment by drakhart** [comment]
Hi Hydreliox,

Does you Super Soco bike use the [old application (Super Soco)](https://play.google.com/store/apps/details?id=com.supersoco.soco) or the [new one (Vmoto Soco)](https://play.google.com/store/apps/details?id=com.shqj.sanye)?

I'm asking because, as of current date, this integration is only compatible with the old application, which is my case (TCmax 2021). I would need access to a new application's user account in order to reverse engineer its API calls and add compatibility.

If that's not the case and you're using the old application, you can see the error logs in Settings > System > Logs. They look similar to this one:

![image](https://user-images.githubusercontent.com/14996002/188497214-9d92068e-fe8a-4d72-b5f7-0dde8514854b.png)

If you wish you can also enable full debug logs by adding these lines into your `configuration.yaml` file and restarting Home Assistant:
```
logger:
  default: info
  logs:
    custom_components.super_soco_custom: debug
```

After that, they should be visible by clicking the "Show full logs" button located at the bottom of the Logs section.

**Comment by HydrelioxGitHub** [comment]
Thank you for your time. I'm using the old app. I will provide logs as soon as possible.

**Comment by drakhart** [comment]
"The only difference is that I can't put a 0 at the begining of the phone number." <- This got me thinking. The phone number field is currently an integer, perhaps it should be a string so it accepts leading zeroes. Honestly, it didn't occur to me that phone numbers could begin with a zero. 😓

I'll try to push a fix as soon as I can.

**Comment by HydrelioxGitHub** [comment]
![20220905_224202.jpg](https://user-images.githubusercontent.com/15862735/188510798-b5fe638d-7a6c-4ea4-bc79-cf0df62dc573.jpg)

In France all cell phone numbers starts with 06 or 07.

**Comment by drakhart** [comment]
Done, this should be fixed in 1.0.0-beta.5. Please let me know if the issue persists.

**Comment by HydrelioxGitHub** [comment]
Ok now I can put a 0 on the front of phone number and it solves the authentication issue.

Thank you very much!
